parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueWriter.java

```diff
@@ -28,4 +30,16 @@
   List<TripleWriter<?>> columns();
 
   void setColumnStore(ColumnWriteStore columnStore);
+
+  /**
+   * Returns a stream of {@link FieldMetrics} that this ParquetValueWriter keeps track of.
+   *
```

Comment:
Javadoc paragraphs need to be separated by `<p>` to display correctly.

Reply:
Thanks for reminding! I'll make sure to double check this in later PRs.